src/templates_parser-print_tree.adb

```diff
@@ -78,6 +81,30 @@ procedure Print_Tree (T : Tree; Level : Natural := 0) is
       Text_IO.Put ((L * 2) * ' ');
    end Print_Indent;
 
+   ---------------
+   -- Sep_Image --
+   ---------------
+
+   function Sep_Image (Str : Unbounded_String) return String is
+      R : Unbounded_String;
+   begin
+      for K in 1 .. Length (Str) loop
```

Comment:
Replacing the 3 lines by
```
for E of To_String (Str) loop
```
would make the code clearer, and potentially more efficient (due to skipping indexing altogether)

Reply:
Done.